BearlyHumans/squirrelville
# Music and ambience triggers

This PR adds two cube trigger areas: MusicTrigger and AmbienceTrigger. Each can be scaled and positioned to fit anywhere on the map. Music and ambient noises can be specified in the respective trigger and continuously loop. Only one music track can play at a time and it fades out before a new track plays. Any number of ambience triggers can be overlapped to layer on several ambient sounds. The music and ambience volume can be adjusted using the music and SFX volume sliders in the options menu.

Steps to test:
1. Be on Jordan scene
2. See where the music and ambient triggers are located in the scene
3. Play the game and walk between the trigger areas
* I even tested if the squirrel spawns in a trigger area

Preview:
* Enable sound of course
* The trigger areas were made visible for demonstration

https://user-images.githubusercontent.com/62001991/133990242-c35c6e01-b1a4-44c4-92cf-9c347fc7c046.mp4
